index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+
+</head>
+<body>
+    <header class="header-section">
+        <a href="" class="logo"> My logo </a>
+<nav class="bttn">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+</nav>
+    </header>
+    <section  >
+        <img class="background-image"src="i-bttn/background-shapes/header-shapes mobile.png" alt="">
+        <p class="text"> <h1>I’m Trevor <br> Glad to see you!</h1></p> 
+        <div><p class="head-section">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
```

Comment:
- [ ]  You should stick to the design as much as possible (e.g. font, colors, images, text, margins), Kindly make sure you update the font of this text, so it looks as much as possible to the Figma design :100: 

Your design

![Screenshot from 2023-01-11 11-30-02](https://user-images.githubusercontent.com/80458199/211769982-1facb771-f2d8-46a3-a0f6-6cea164b26af.png)

Figma design

![Screenshot from 2023-01-11 11-30-27](https://user-images.githubusercontent.com/80458199/211770071-3b400ecc-42f7-41dc-9408-29a71992b1a0.png)